frontend/src/components/ProvideSQLEditorContext.vue

```diff
@@ -125,45 +127,44 @@ const prepareSheet = async () => {
     },
   });
 
-  // TODO(Jim): remove this after refactor `sharedSheet`
-  useSQLEditorStore().setSQLEditorState({
-    sharedSheet: sheet,
-  });
-
   return true;
 };
 
 const prepareConnectionSlug = async () => {
-  // TODO(Jim): use standard slug format instead of "_" joint format.
-
   const connectionSlug = (route.params.connectionSlug as string) || "";
-  // [instanceName, instanceId, databaseName, databaseId]
-  const parts = connectionSlug.split("_");
-  const instanceId = parseInt(parts[1], 10);
-  const databaseId = parseInt(parts[3], 10);
-  if (Number.isNaN(instanceId) || Number.isNaN(databaseId)) {
+  const [instanceSlug, databaseSlug = ""] = connectionSlug.split("_");
```

Comment:
It's more readable to co-locate this with connectionSlug and comment the rationale